crates/hc_bundle/tests/test_cli.rs

```diff
@@ -283,6 +283,20 @@ async fn test_multi_integrity() {
     assert_eq!(*dna.dna_def(), expected);
 }
 
+#[test]
+fn test_hash_dna_function() {
+    {
+        let mut cmd = Command::cargo_bin("hc-dna").unwrap();
+        let cmd = cmd.args(["hash", "tests/fixtures/my-app/dnas/dna1"]);
+        cmd.assert().success();
```

Comment:
It would be good to assert the output length here if the dna hashes can change. Then we can be reasonably confident that the command is doing the right thing without risking having brittle tests

Reply:
additional test added